bindings/hyperdrivepy/src/hyperdrive_state_methods/lp/add.rs

```diff
@@ -98,9 +105,23 @@ impl HyperdriveState {
                     err
                 ))
             })?;
-        let result1 = U256::from(result_fp1).to_string();
+        let result1 = U256::try_from(result_fp1)
+            .map_err(|err| {
+                PyErr::new::<PyValueError, _>(format!(
+                "Failed to convert calculate_pool_deltas_after_add_liquidity.0 (share_reserves) result {} to U256: {}",
```

Comment:
again, since this is going to be seen in a python environment I think `[0]` notation makes more sense.